lib/pymedphys/_experimental/streamlit/apps/sum_doses.py

```diff
@@ -51,34 +51,36 @@ def main():
         st.write(e)
         st.stop()
 
-    if not datasets:
-        st.stop()
+    if st.button("Click to Sum Doses"):
 
-    with right_column:
-        st.write(
-            f"""
+        with right_column:
+            st.write(
+                f"""
 
-            ## Details
+                ## Details
 
-            * Patient ID: `{datasets[0].PatientID}`
-            * Patient Name: `{get_pretty_patient_name_from_dicom_dataset(datasets[0])}`
-            """
-        )
+                * Patient ID: `{datasets[0].PatientID}`
+                * Patient Name: `{get_pretty_patient_name_from_dicom_dataset(datasets[0])}`
+                """
+            )
 
-    if len(datasets) < 2:
-        st.stop()
+        if len(datasets) < 2:
+            # Note that if the user tries to remove a file that has
```

Comment:
This note is something to be aware of. It's actually quite annoying...

Haven't tried working around it yet